lib/superstore/types/range_type.rb

```diff
@@ -22,7 +22,9 @@ def deserialize(range_tuple)
       end
 
       def cast_value(value)
-        if value.is_a?(Range) && (value.end.nil? || value.begin <= value.end)
+        if is_beginless_date_range?(value)
```

Comment:
We don't really have a need for beginless date ranges. Is 2.7.0 broken without this?

Reply:
Yeah, that's what I was thinking. They seem pretty senseless, which is why that method will just return nil in the case of one.

And the 2.7 build does fail on this test which is checking that a range without a start returns nil https://github.com/data-axle/superstore/blob/master/test/unit/types/date_range_type_test.rb#L26. (In the past, it returned nil by being caught in the ArgumentError rescue due to how Ruby handled ranges with a nil beginning in 2.6).

Reply:
I see. Makes sense.